src/core/assets/copy/copy.ts

```diff
@@ -0,0 +1,23 @@
+import { type IOdsCoreCopy } from './IOdsCoreCopy';
```

Comment:
If we apply the suggestion above, I'd rename this file to `coreCopy.ts` (or `enCopy.ts` if we keep it named as `enCopy`)

Reply:
✅ https://github.com/aragon/ods/pull/221/commits/e5e0ecb68c8c0f30b1055fb3c24e8e9b02be13a6